tests/app/views/test_brief_response.py

```diff
@@ -29,10 +29,27 @@ def setup(self):
                 supplier_id=0,
             )
 
-            db.session.add_all([service, brief])
+            specialist_brief = Brief(
+                data=example_listings.brief_data().example(),
+                status='live', framework_id=5, lot=Lot.query.get(6)
```

Comment:
Relying on existing framework and lot IDs in tests feels wrong to me, but we do it all over the place and fixing it isn't part of this PR... another one to chalk up for "after DOS live" though.